staff/sergiomons/css-layout-simple-adaptive/styles/main.css

```diff
@@ -0,0 +1,2 @@
+@import "mobile.css" screen and (max-width: 768px);
+@import "desktop.css" screen and (min-width: 768px);
```

Comment:
What are the common properties that are repeated in both files (mobile and desktop)? Should you group them here?

Reply:
Yes, you're right! Thanks!

Reply:
I have modified it.